src/cltools/Benchmark.cpp

```diff
@@ -255,6 +261,173 @@ struct Kernel :
   }
 };
 
+namespace benchDistributions {
```

Comment:
If this is only used here, I think it's better to use an anonymous namespace to avoid the risk of clashes

Reply:
Why? I think you just have to write
```
std::unique_ptr<AtomDistribution> distribution
```

I think we should avoid to add custom namespace because at some point there will be a name clash. If it's only in a cpp file, anonymous is usually the right thing to do.

Reply:
Made it anonymus, I also changed the name of the factory function, so that is more clear